backend/src/main/java/com/javahelp/backend/accountdeletion/DeleteManager.java

```diff
@@ -0,0 +1,38 @@
+package com.javahelp.backend.accountdeletion;
```

Comment:
You might want to move all of these into a package like `com.javahelp.backend.domain.user.delete`